framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
Should definitively not be approved until others shared their thoughts - since it changes established behaviour.

Another solution would be to document the implications that using Command has on the UI consistency and shed light on the posibility to trigger disabled features.